lib/alephant/broker/load_strategy/revalidate/refresher.rb

```diff
@@ -0,0 +1,62 @@
+module Alephant
+  module Broker
+    module LoadStrategy
+      module Revalidate
+        class Refresher
+          include Logger
+
+          attr_reader :component_meta
+
+          def initialize(component_meta)
+            @component_meta = component_meta
+          end
+
+          def refresh
+            return if cache.get(inflight_cache_key)
+
+            queue.send_message(component_meta.to_json)
+            cache.set(inflight_cache_key, true)
+          end
+
+          private
+
+          def queue
+            @queue ||= begin
```

Comment:
yeah for us, we've only ever used `begin` when we know we're going to have to handle an error and `rescue` (although we also typically use the `def/rescue` style), so this version of using `begin` without a rescue just seems odd to me.

Also the actual Ruby semantics of `begin` is for defining code before the program/file itself is executed, where as Procs/Lambdas are defined as arbitrary code that is executed when needed (which semantically fits more cleanly here IMHO).

But I'm not overly precious about which route you prefer to take, although I could see there being a code consistency concern considering we've not utilised it any where else in this fashion. But that doesn't mean it can't go that way if we wanted it to either.